src/Phinx/Console/Command/Create.php

```diff
@@ -131,7 +194,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // Do we need to do the post creation call to the creation class?
+        if (!empty($creationClassName)) {
+            try {
+                $creationClass->postMigrationCreation($filePath, $className, $this->getConfig()->getMigrationBaseClassName());
+            } catch(\Exception $ex) {
+                throw new \RuntimeException(sprintf(
+                    'Problem calling %s->postMigrationCreation(), resulting in %s',
```

Comment:
Not sure if this is worth a try/catch block... it might actually be better to just allow the exception to bubble, so that debugging is more clear.